README.rst

```diff
@@ -343,10 +374,71 @@ Remove a country from a database:
+
+To run the test suite:
+
+#. Install Poetry if you haven't already. Visit the Poetry website for `installation instructions <https://python-poetry.org/docs/#installation>`_.
+#. Install the project dependencies by running ``poetry install`` in the project's root directory. This command also installs necessary tools like ``mypy`` and ``pre-commit`` as defined in ``pyproject.toml``.
+#. Activate the Poetry shell with ``poetry shell``. This will spawn a new shell subprocess, which is configured to use your project’s virtual environment.
+#. Run the unit tests using ``make test``. Ensure you have ``make`` installed on your system (commonly pre-installed on Unix-like systems).
+#. Run type checks using ``mypy .`` to ensure type consistency.
+#. Run linting checks using ``pre-commit run --all-files`` to verify code formatting and style.
+#. Ensure all tests pass successfully.
+
+If you add new features or fix bugs, please include corresponding tests. Follow the project's coding standards and update documentation as needed.
+
+Note: The project's dependencies and the environment needed to run tests are managed by Poetry, using the ``pyproject.toml`` and ``poetry.lock`` files.
+
+License
+-------
+
+``pycountry`` is made available under the GNU Lesser General Public License Version 2.1 (LGPL 2.1). This license allows you to use, modify, and distribute the library in your own projects.
+
+For more details, see the `LICENSE <https://github.com/pycountry/pycountry/blob/main/LICENSE.txt>`_ file included with the source code.
+
+Credits
+-------
+
+``pycountry`` is developed and maintained by a community of developers and contributors. Special thanks to everyone who has contributed their time and effort.
+We gratefully acknowledge the Debian `pkg iso-codes <https://salsa.debian.org/iso-codes-team/iso-codes>`_ team and contributors for their work and for making this resource freely available.
+
+For a complete list of contributors, see the `COPYRIGHT <https://github.com/pycountry/pycountry/blob/main/COPYRIGHT.txt>`_ file.
+
+Acknowledgments
+----------------
+
+We would like to express our gratitude to the authors and maintainers of the following libraries, which have greatly contributed to the functionality and internationalization of ``pycountry``:
+
+* `country-info <https://github.com/countryinfo/countryinfo>`_
+* `babel <https://github.com/python-babel/babel>`_
+
+These libraries provide valuable data and localization support that complement the features of ``pycountry``.
+
+Maintainers
+-----------
 
-Some users have reported issues using PyCountry with PyInstaller guidance on
-how to handle the issues can be found in the `PyInstaller Google Group
-<https://groups.google.com/g/pyinstaller/c/OYhJdeZ9010/m/vLhYAWUzAQAJ>`_.
+* `Christian Theune <mailto:ct@flyingcircus.io>`_
+* `Nate Schimmoller <mailto:nschimmo@gmail.com>`_
+* `Zachary Ware <mailto:zachary.ware@gmail.com>`_
```

Comment:
Better to link our GitHub handles than our email addresses, I think.

Reply:
Is that just using @ or is there a better way to link?

Reply:
Done!